src/main/resources/db_scripts/db_script_v3.sql

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Careful with the version number. I would already increase it to v4.

Reply:
Yes, I know. It will be showed as a conflict

Reply:
Solved